x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/jinaai/JinaAIServiceSettings.java

```diff
@@ -83,7 +86,7 @@ public JinaAIServiceSettings(StreamInput in) throws IOException {
 
     // should only be used for testing, public because it's accessed outside of the package
     public JinaAIServiceSettings() {
```

Comment:
It doesn't look like this constructor is used, I think I'd prefer if we removed it that way we don't accidentally try to create a default service settings with a default model id.